src/components/Pages/Feedback/FeedbackForm.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import styled from 'styled-components';
+
+const StyledForm = styled.form`
```

Comment:
Form looks really good. You can abstract out the styling into a style.js file if you're feeling fancy.